src/app/htmltest.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Don't think this file is being used as it's called `htmltest.html` ?

Reply:
removed